### PR TITLE
Adds new documentation for view helpers

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Enhancement        | yes

### CODE REVIEW

**Comment by froschdesign** [comment]
Related to #1

File: docs/book/view-helpers/translate-plural.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+of "car", for one car, and the plural definition, "cars", meaning zero "cars"
+or more than one car. Other languages like Russian or Polish have more plurals
+with different rules.
+
+The helper acts as a wrapper for the `Laminas\I18n\Translator\Translator` class.
+
+## Setup
+
+Before using the `TranslatePlural` view helper, you must have first created a
+`Translator` object and have attached it to the view helper. If you use the
```
Comment (michalbundyra) [inline docs/book/view-helpers/translate-plural.md]: Do we have documentation how to create Translation object (if not using HelperPluginManager?

File: docs/book/view-helpers/translate-plural.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+(The above example assumes that the environment locale is set to `de_DE`.)
+
+## Using Text Domain
+
+The text domain defines the domain of the translation.
+
+```php fct_label="Invoke Usage"
+echo $this->translatePlural('monitor', 'monitors', 1, 'customDomain'); // 'Monitor'
```
Comment (michalbundyra) [inline docs/book/view-helpers/translate-plural.md]: I am not sure if this example is good - it looks like just capitalise first letter, would be clearer to use car <-> Auto as in example below, imo.

File: docs/book/view-helpers/translate-plural.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+```php
+echo $this->translatePlural('car', 'cars', 1, 'default', 'de_DE'); // 'Auto'
+echo $this->translatePlural('car', 'cars', 4, 'default', 'de_DE'); // 'Autos'
+```
+
+### Default Value
+
+By default, if no locale is provided, `TranslatePlural` will use the system
+locale provide by PHP's `Locale::getDefault()`.
```
Comment (michalbundyra) [inline docs/book/view-helpers/translate-plural.md]: provide -> provided

File: docs/book/view-helpers/translate.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+The locale to which the message should be translated.
+
+```php
+echo $this->translate('car', 'default', 'de_DE'); // 'Auto'
+```
+
+### Default Value
+
+By default, if no locale is provided, `TranslatePlural` will use the system
+locale provide by PHP's `Locale::getDefault()`.
```
Comment (michalbundyra) [inline docs/book/view-helpers/translate.md]: provide -> provided

File: docs/book/view-helpers/plural.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Therefore we often need to handle plural cases even without translation
+(mono-lingual application). The `Plural` helper was created for this.
+
+> ### Plural Helper does not translate
+>
+> If you need to handle both plural cases *and* translations, you must use the
+> [`TranslatePlural` helper](translate-plural.md); `Plural` does not translate.
+
+Internally, the `Plural` helper uses the `Laminas\I18n\Translator\Plural\Rule` class to handle rules.
```
[the remark this answers is not in the document]
Comment (froschdesign) [reply docs/book/view-helpers/plural.md]: The topic plural messages will be covered in a separated chapter/page.

File: docs/book/view-helpers/translate-plural.md (hunk 1 above)
Comment (froschdesign) [reply docs/book/view-helpers/translate-plural.md]: The topic will be covered in a separated chapter/page.

File: docs/book/view-helpers/plural.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (froschdesign) [reply docs/book/view-helpers/plural.md]: See: https://github.com/laminas/laminas-i18n/projects/1

File: docs/book/view-helpers/translate-plural.md (hunk 1 above)
Comment (froschdesign) [reply docs/book/view-helpers/translate-plural.md]: See: https://github.com/laminas/laminas-i18n/projects/1

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Comment by froschdesign** [comment]
@michalbundyra 
Thanks for your support! 👍

**Pull request merged**